cmplibrary/src/androidTest/java/com/sourcepoint/cmplibrary/data/local/DataStorageCcpaImplTest.kt

```diff
@@ -17,21 +17,21 @@ class DataStorageCcpaImplTest {
     fun check_DataStorage_TcData_gets_stored() {
         // Context of the app under test.
         val appContext = InstrumentationRegistry.getInstrumentation().targetContext
-        val storage = DataStorageCcpa.create(appContext).apply { clearAll() }
+        val storage = DataStorageCcpa.create(appContext).apply { deleteCcpaConsent() }
         val map = TreeMap<String, String>()
     }
 
     @Test
     fun clear_data_DataStorage() {
```

Comment:
I think we should rename the test case name to something more descriptive of what the test is actually doing, something like
> given_delete_ccpa_consent_called_with_stored_consent_then_should_clear_stored_consent

or something along those lines

Reply:
@carmelo-iriti was this addressed?